tide/master/MasterApplication.cpp

```diff
@@ -503,7 +504,12 @@ void MasterApplication::_initRestInterface()
                 _logger.get(), &LoggingUtility::powerStateChanged);
 
         connect(_restInterface.get(), &RestInterface::powerOff, [this]() {
-            if (!_planarController->powerOff())
+            if (_planarController->powerOff())
+            {
+                DisplayGroupController controller(*_displayGroup.get());
```

Comment:
no .get() on smart ptrs!! also can be on a single line: DisplayGroupController(*_displayGroup).hidePanels();